demisto_sdk/commands/test_content/test_modeling_rule/test_modeling_rule.py

```diff
@@ -145,9 +153,11 @@ def verify_results(
         # get expected_values for the given query result
         td_event_id = result.pop(f"{tested_dataset}.test_data_event_id")
         expected_values = None
+        tenant_timezone: str = ""
         for e in test_data.data:
             if str(e.test_data_event_id) == td_event_id:
                 expected_values = e.expected_values
+                tenant_timezone = e.tenant_timezone
                 break
 
```

Comment:
do we _always_ get the tenant_timezone, after this loop is done?

Reply:
Basically yes (just like `expected_values`) since the `tenant_timezone` is in the root such as `test_data_event_id`
![Screenshot 2023-07-05 at 15 18 50](https://github.com/demisto/demisto-sdk/assets/78307768/0a6d10e9-0f87-4f27-ac83-b4c4279a7702)

Reply:
so the empty-string default could cause trouble down the road.
Please add `if not tenant_timezone: raise ValueError("could not find timezone")`, or at least `logger.warning`

Reply:
@dorschw 
I can set the default to UTC

Reply:
this may go unnoticed as well